### PR TITLE
test: Create integration tests for DID URL dereferencing [DEV-2519]

### PR DESCRIPTION
**Pull request by abdulla-ashurov**
- [x] Added integration tests for testing different DIDDoc query parameters
- [x] Added integration tests for testing different Resource query parameters
- [x] Added integration tests for testing combination of DIDDoc and Resource query parameters

### CODE REVIEW

File: tests/integration/rest/diddoc/query/service/negative_test.go [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+			},
+			ExpectedStatusCode: types.NotFoundHttpCode,
+		},
+	),
+
+	Entry(
+		"cannot redirect to serviceEndpoint with relativeRef query parameter",
+		utils.NegativeTestCase{
+			DidURL: fmt.Sprintf(
+				"http://localhost:8080/1.0/identifiers/%s?relativeRef=/info",
```
Comment (anikitinDSR) [inline tests/integration/rest/diddoc/query/service/negative_test.go:64]: Could you change `/info` to `$2Finfo` cause it requires to use URL encoding.

File: tests/integration/rest/diddoc/query/service/negative_test.go [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+			},
+			ExpectedStatusCode: types.RepresentationNotSupportedHttpCode,
+		},
+	),
+
+	Entry(
+		"cannot redirect to serviceEndpoint with an existent service, but an invalid relativeRef URI parameters",
+		utils.NegativeTestCase{
+			DidURL: fmt.Sprintf(
+				"http://localhost:8080/1.0/identifiers/%s?relativeRef=/info",
```
Comment (anikitinDSR) [inline tests/integration/rest/diddoc/query/service/negative_test.go:90]: the same

**Review by anikitinDSR — APPROVED** [review]

File: tests/integration/rest/diddoc/query/service/negative_test.go (hunk 1 above)
Comment (abdulla-ashurov) [reply tests/integration/rest/diddoc/query/service/negative_test.go]: Done! ✅

File: tests/integration/rest/diddoc/query/service/negative_test.go (hunk 2 above)
Comment (abdulla-ashurov) [reply tests/integration/rest/diddoc/query/service/negative_test.go]: Done! ✅

**Review by anikitinDSR — APPROVED** [review]

**Pull request merged**